themes/src/main/resources/theme/keycloak.v2/login/field.ftl

```diff
@@ -88,6 +88,54 @@
   </@group>
 </#macro>
 
+<#macro clipboard name label value="" readonly=true>
+  <@group name=name label=label>
+    <div class="pf-v5-c-clipboard-copy" id="${name}-clipboard">
```

Comment:
Like other classes, e.g. `kcInputGroupItemClass`, this class should be provided by a property to allow easy customization. This applies for all `pf-` classes.